authorization-common/src/main/java/com/sequenceiq/authorization/service/list/AuthorizationResourceProvider.java

```diff
@@ -0,0 +1,7 @@
+package com.sequenceiq.authorization.service.list;
+
+import java.util.List;
+
+public interface AuthorizationResourceProvider {
```

Comment:
so in authorization-common we have `ResourceNameProvider`, `ResourceBasedCrnProvider`, `ResourceNameAndCrnProvider`, `ResourceCrnAndNameProvider` and now we have this, at this point we need to rethink these providers to avoid confusion